src/main/scala/uk/gov/hmrc/agentmtdidentifiers/model/Vrn.scala

```diff
@@ -16,7 +16,13 @@
 
 package uk.gov.hmrc.agentmtdidentifiers.model
 
-object VrnValidation {
+import uk.gov.hmrc.domain.{SimpleObjectReads, SimpleObjectWrites, TaxIdentifier}
+
+case class Vrn(value: String) extends TaxIdentifier
+
+object Vrn {
+  implicit val arnReads = new SimpleObjectReads[Arn]("value", Arn.apply)
```

Comment:
Should it be ```Vrn```?

Reply:
Yep, I somehow forgot to commit a couple of files.